rotkehlchen/tests/fixtures/exchanges/okx.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+from rotkehlchen.tests.utils.exchanges import create_test_okx
+from rotkehlchen.types import ApiKey, ApiSecret
+
+
+@pytest.fixture
+def mock_okx(
+        database,
+        inquirer,  # pylint: disable=unused-argument
+        function_scope_messages_aggregator,
+):
+    api_key = ApiKey('f32f48d7-74ad-41ce-8028-fcc4e4589f9c')
+    secret = ApiSecret(b'3DC350723E8200C236792784644E17A0')
+    passphrase = 'Rotki123!'
```

Comment:
For this I'm seeing and in the other exchanges we create fake api keys for this except for gemini that we use the sandbox. Since you are mocking the requests in which test are the keys needed?

https://github.com/rotki/rotki/blob/d8e16391d5c35b471eebb0635d4334a248d06aae/rotkehlchen/tests/utils/exchanges.py#L802

Reply:
OKX API requires authentication for the currencies endpoint, used in `test_assets_are_known`

Reply:
Yes, I've checked the API and there is no way around this it seems.